spec/oncocore.txt

```diff
@@ -8,22 +8,40 @@ CodeSystem:     LNC = http://loinc.org
 CodeSystem:	    MTH = http://ncimeta.nci.nih.gov
 CodeSystem:     UCUM = http://unitsofmeasure.org
 
+Abstract Element:	CancerCondition
+Based on:		Condition
+Description:	"Abstract class for describing a primary or secondary metastatic neoplastic diseases."
+				PersonInformationSource is type PractitionerInformationSource
+0..1            HistologyMorphologyBehavior
+				BodyLocation.LocationCode from CancerBodyLocationVS if covered
+0..0            Severity
 
-EntryElement:   CancerCondition
-Based on:	    Condition
-Description:    "The presence of malignant neoplastic disease, in which abnormal cells divide without control and can invade nearby tissues."
-                Code from CancerDisorderVS if covered // TODO: Document reasons to used SNOMED as opposed to ICD-O-3
-//                StageInformation.StageSummary from http://loinc.org/vs/LL240-3   // Commented out because LL240-3 only applies to TNM stage group - is this correct?
+EntryElement:   PrimaryCancerCondition
+Based on:	    CancerCondition
+Description:    "Primary cancer condition, in which abnormal cells divide without control and can invade nearby tissues."
+                Code from PrimaryCancerDisorderVS if covered
   	            StageInformation.StageDetail value is type ref(CancerStageGroup)
+
+	Element:		HistologyMorphologyBehavior
+	Description:	"A description of the morphologic and behavioral characteristics of the cancer."
+	Value:			CodeableConcept should be from HistologyMorphologyBehaviorVS
+
+EntryElement:   SecondaryCancerCondition
+Based on:       Condition
+Description:    "Records the history of secondary neoplasms, including location(s) and the date of onset of metastases."
+                Code from SecondaryCancerConditionVS
+0..1			RelatedCancerCondition
```

Comment:
Should this be constrained to `PrimaryCancerCondition` or `Tumor`?

Reply:
That's probably true. Added "RelatedCancerCondition value is type PrimaryCancerCondition"